cli/Tests/TuistCASTests/CASServiceTests.swift

```diff
@@ -132,6 +133,9 @@ struct CASServiceTests {
             #expect(Bool(false), "Expected success, but contents is nil")
         }
 
+        // Wait for background task to complete
+        try await Task.sleep(for: .milliseconds(100))
```

Comment:
this is obviously not ideal, but this is a quick way for us to still be able to assert the fire-and-forget operation.

We don't have a lot of these, so I'd argue this is fine for the time being.